src/inc/internal/AppxBundleWriter.hpp

```diff
@@ -0,0 +1,82 @@
+//
+//  Copyright (C) 2019 Microsoft.  All rights reserved.
+//  See LICENSE file in the project root for full license information.
+// 
+#pragma once
+
+#include "AppxPackaging.hpp"
+#include "ComHelper.hpp"
+#include "DirectoryObject.hpp"
+#include "AppxBlockMapWriter.hpp"
+#include "ContentTypeWriter.hpp"
+#include "ZipObjectWriter.hpp"
+#include "BundleWriterHelper.hpp"
+#include "BundleManifestWriter.hpp"
+#include "AppxPackageInfo.hpp"
+
+#include <map>
+#include <memory>
+#include <future>
+

```

Comment:
Make sure to only include headers when you actually need them

Reply:
Removed these headers.